tests/e2e/TextSearch/TextBoxSearch.spec.ts

```diff
@@ -1,40 +1,41 @@
 import page from "./pages/home.page";
-import { Element } from "webdriverio";
 
 const testValue = "Uganda";
 
 describe("Text box search", () => {
     it("when query is entered in the search input the list view should be filtered", () => {
         page.open();
-        page.searchInput.waitForVisible();
+        page.searchInput.waitForDisplayed();
+        expect(page.listViewListItems.length).toBe(5);
         page.searchInput.click();
         page.searchInput.setValue(testValue);
-
-        const listviewItems: Element[] = page.listViewList.value;
-        expect(listviewItems.length).toBe(1);
+        browser.pause(1000);
+        expect(page.listViewListItems.length).toBe(1);
     });
 
     it("when query is entered in the search input and clear button clicked the list view should be filtered with new query", () => {
```

Comment:
It's good to check whether the query send after using the clear button is the correct one by verifying that the total list items at the beginning of the test is also 5.

Reply:
That is done on line 27, `searchButton` should be renamed to clearButton

Reply:
done